docs/modules/ROOT/pages/how-tos/upgrade.adoc

```diff
@@ -0,0 +1,109 @@
+= How to Upgrade cert-manager
+
+== Upgrade `component-cert-manager` from `v1.0.0` to `v2.0.0`
+
+This upgrade upgrades the underlying `cert-manager` helm chart from `v0.15.1` to `v1.4.0`. There are some breaking changes in `cert-manager`.
+
+https://github.com/jetstack/cert-manager/releases/tag/v0.16.0
+
+- Support for AuditSink resources in the auditregistration.k8s.io/v1alpha1 API group has been removed
+
+https://github.com/jetstack/cert-manager/releases/tag/v1.2.0
+
+- The minimum supported Kubernetes version is now v1.16.0 as of cert-manager v1.2.0. Users still running Kubernetes v1.15 or below should upgrade to a supported version before installing cert-manager or use cert-manager v1.1.
+- The User-Agent request header sent by cert-manager has changed to reflect the ownership transfer to the CNCF
+- The --renew-before-expiration-duration flag of the cert-manager controller-manager has been deprecated. Please set the Certificate.Spec.RenewBefore field instead. This flag will be removed in the next release.
+Certificates issued by the Vault issuer have changed — the root CA instead of the issuing CA is now stored in ca.cr
+
+https://github.com/jetstack/cert-manager/releases/tag/v1.3.0
+
+- CertificateRequests are now immutable - the spec and metadata.annotations fields cannot be changed after creation. They were always designed to be immutable but this behavior is now enforced by the cert-manager webhook.
+- This release updates the Venafi Cloud Issuer to use OutagePREDICT instead of DevOpsACCELERATE.
+
+https://github.com/jetstack/cert-manager/releases/tag/v1.4.0
+
+- The CA issuer now attempts to store the root CA instead of the issuing CA into the ca.crt field for issued certificates; this is a change of behavior. All of the information which was previously available is still available: the intermediate should appear as part of the chain in tls.crt.
+- RunAsNonRoot is now enabled by default in the securityContext values. If you're using custom containers with the chart that run as root, you will need to set this back to false.
```

Comment:
It's not necessary to copy-paste the entire changelog of cert-manager itself. It's enough to link to the changelog upstream I'd say.